projects/backend/index.ts

```diff
@@ -3,11 +3,13 @@ require('dotenv').config()
 import awsServerlessExpress from 'aws-serverless-express'
 import { Handler } from 'aws-lambda'
 import express = require('express')
-import { issueController } from './controllers/issue'
+import { issueController, issuesSummaryController } from './controllers/issue'
 import { frontController, collectionsController } from './controllers/fronts'
 
 const app = express()
 
+app.get('/issue', issuesSummaryController)
```

Comment:
Can we change this to `issues` just so it's a touch clearer? Also, am just about to merge a change that will convert these paths to use route builders rather than raw string, but I'm happy to change that when it goes it 🆗

Reply:
Will do.